content/_code-samples/escrow/py/create_escrow.py

```diff
@@ -0,0 +1,46 @@
+from xrpl.wallet import Wallet, generate_faucet_wallet
+from xrpl.utils import xrp_to_drops
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.models import EscrowCreate                             
+
+# Create Escrow
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # instanstiate ripple client
+
+amount = 10.000 # amount to escrow
+
+receiver_addr = generate_faucet_wallet(client=client).classic_address # receiver address
```

Comment:
I suggest instead of generating and funding an example wallet that will never be used again, use the Testnet XRP Faucet address (`rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe`) as the example recipient. That way when the escrow is finished, the XRP is returned to the faucet and can be recycled, instead of filling up the Testnet with more unused accounts.

```suggestion
receiver_addr = "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe" # Example: send back to Testnet Faucet
```